src/main/java/com/machowina/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.machowina.exception;
+
+public class EntityNotFoundException extends RuntimeException{
+
+	private static final long serialVersionUID = 4681194719623412061L;
```

Comment:
Po co ta stałą?

Reply:
Eclipse podkreślał mi na żółto że powinnam wygenerować coś takiego, więc na wszelki wypadek go posłuchałam.
Z tego co rozumiem jest to używane do serializacji, więc chyba mi niepotrzebne